exchanges/gemini/gemini_wrapper.go

```diff
@@ -619,9 +669,43 @@ func (g *Gemini) CancelAllOrders(ctx context.Context, _ *order.Cancel) (order.Ca
 }
 
 // GetOrderInfo returns order information based on order ID
-func (g *Gemini) GetOrderInfo(ctx context.Context, orderID string, pair currency.Pair, assetType asset.Item) (order.Detail, error) {
-	var orderDetail order.Detail
-	return orderDetail, common.ErrNotYetImplemented
+func (g *Gemini) GetOrderInfo(ctx context.Context, orderID string, _ currency.Pair, _ asset.Item) (order.Detail, error) {
```

Comment:
I viewed it as an annoyance to know the asset and currency if you just had an orderID and wanted to get the order. But I'll re-evaluate this function, may also make `order.Detail` a pointer response

Reply:
What I've done is add validation to asset, currency pair when its used in the function already, but I haven't gone beyond that.

I think it would be good to change this to have say a `*order.Request` type which could include the ClientOrderID or potentially some other fields with more validation, but I'm going to say that's for another time.

Let me know if you want me to do more with it though

Reply:
Nar that's fine this PR is already a monster. Thanks!